taskcluster/ac_taskgraph/loader/build_config.py

```diff
@@ -49,7 +49,7 @@ def get_upstream_deps_for_components(components):
     for configuration in CONFIGURATIONS_WITH_DEPENDENCIES:
         logger.info("Looking for dependencies in '%s' configuration" % configuration)
 
-        cmd = ["./gradlew"]
+        cmd = ["./gradlew", "--console=plain", "--parallel"]
```

Comment:
I followed your findings regarding parallel, @gbrownmozilla. I put `--console=plain` in case something changes in the gradle output in the future. We want the the output as raw as possible since we parse it.

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1788606#c3